examples/http/client.rb

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+require 'rubygems'
+require 'bundler/setup'
+require 'faraday'
+# Require otel-ruby
+require 'opentelemetry'
+require 'opentelemetry/sdk'
+
+require_relative '../exporters/console'
+
+# Allow setting the host from the ENV
+host = ENV.fetch('HTTP_EXAMPLE_HOST', '0.0.0.0')
+
+SDK = OpenTelemetry::SDK
+OpenTelemetry.tracer_factory = SDK::Trace::TracerFactory.new
+
+# Configure tracer
+exporter = Examples::Exporters::Console.new
+processor = SDK::Trace::Export::SimpleSpanProcessor.new(exporter)
+tracer = OpenTelemetry.tracer_factory.tracer('http.client', 'semver:1.0')
+tracer.add_span_processor(processor)
+
+connection = Faraday.new("http://#{host}:4567")
+url = '/hello'
+
+# For attribute naming, see:
+# https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-semantic-conventions.md#http-client
+
+# Span name should be set to URI path value:
+tracer.in_span('/hello', kind: 'Client') do |span|
```

Comment:
`kind:` should be `OpenTelemetry::Trace::SpanKind::CLIENT`, which really rolls off the tongue 😞. We should consider whether we just want to support symbols directly here, like:
```ruby
:client
:server
:internal
:producer
:consumer
```

Reply:
@fbogsany can do! I didn't realize Span kind was limited to a list of values. Funny timing on [this spec issue](https://github.com/open-telemetry/opentelemetry-specification/pull/337)

Reply:
_Note: we can use the symbols after #134_